historyserver/pkg/collector/logcollector/runtime/logcollector/collector.go

```diff
@@ -632,6 +676,26 @@ func (r *RayLogHandler) processSessionPrevLogs(sessionDir string) {
 	}
 }
 
+// isFileAlreadyPersisted checks if a file has already been persisted to persist-complete-logs
```

Comment:
`isFileAlreadyPersisted` detects files already moved to persist-complete-logs and avoid duplicate uploads.